ci/travis/test_script.sh

```diff
@@ -30,7 +30,7 @@ if [[ $BUILD_DOCS == false ]]; then
   fi
 else
   cd doc
-  python make.py html -n 2
+  python make.py html
```

Comment:
I I have definitely seen good speed up on building the docs (as they are now) but have found you need to use ~2x the number of workers compared to the number of core.